### PR TITLE
Create a basic webpack config

### PR DESCRIPTION
**Pull request by vujita**

### CODE REVIEW

**Comment by alanhamlett** [comment]
It doesn't seem to be compiling the typescript correctly.

File: config/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/**
```
Comment (alanhamlett) [inline config/webpack.config.js]: let's put webpack.config.js in the root folder... not enough configs for a `config` folder.

Comment (vujita) [inline config/webpack.config.js]: what about any other configs that may arise like karma or jasmine?

Comment (alanhamlett) [inline config/webpack.config.js]: I prefer they all stay visible.

**Comment by alanhamlett** [comment]
Wohoo thanks!

**Pull request merged**